samplelistener/listener/listener.bal

```diff
@@ -1,24 +1,30 @@
 import ballerina/http;
 
-class SlackListener {
+class Listener {
    private http:Listener httpListener;
     private DispatcherService dispatcherService;
-   //private string verificationToken;
 
-   public isolated function init() returns error? { //receive verification token with listener config
-        self.httpListener = check new (8090); // param port
-        //self.verificationToken = userConfig.verificationToken
+   public isolated function init(int port = 8090) returns error? { 
+        self.httpListener = check new (port);
+        self.dispatcherService = check new DispatcherService();
    }
 
-   public isolated function attach(SlackAppMentionHandlingService|SlackAppCreatedHandlingService serviceRef, () attachPoint) returns @tainted error? {// Pass verification token 
-        check self.httpListener.attach(check new DispatcherService(serviceRef), attachPoint);
+   public isolated function attach(GenericService serviceRef, () attachPoint) returns @tainted error? {
```

Comment:
Shall make type of attachPoint string | string [] ? So we can pass the different context paths when we have multiple services using single listener

Reply:
Are we expecting users to specify the path? How do we dispatch based on the path?

Reply:
Thank you so much for the feedback. 

Even though, the users can use multiple services on a single listener, we are using only one dispatcher service for a listener internally. All the received events will be directed to the path `/events` in the dispatcher service and then the events will be redirected to the user defined remote functions. Hence, IMO, there is no use of letting the user to define the path in this approach.